api/src/paths/user/self.test.ts

```diff
@@ -64,7 +64,7 @@ describe('getUser', function () {
       const result = self.getUser();
 
       // eslint-disable-next-line
-      //@ts-ignore
```

Comment:
If we cast the value to an `any` does it solve this issue (or possibly the unknown/any thing)?  Might be nicer than ignores everywhere?

Reply:
the issue is with it missing like 76 fields that I dont need to mock out

Reply:
Should be able to do something like:

```Typescript
  const sampleReq = {
    keycloak_token: {}
  } as any;
    
  await result(sampleReq, (null as unknown) as any, (null as unknown) as any);
```

And then won't need any of the ignores

Reply:
thanks!